packages/lucy/src/pocket.js

```diff
@@ -0,0 +1,97 @@
+export default function (part) {
+  const { options, Point, Path, points, paths, Snippet, snippets, complete, sa, paperless, macro } =
+    part.shorthand()
+
+
+    //pocket seams here 
+  let pocketLength = (800 * options.length)
+  let pocketWidth = (500 * options.width)
+  let pocketEdge = (options.edge)

```

Comment:
```suggestion
  let pocketLength = 800 * options.length
  let pocketWidth = 500 * options.width
  let pocketEdge = options.edge

```
Unnecessary brackets